pubspec.yaml

```diff
@@ -37,6 +37,7 @@ dev_dependencies:
   json_serializable: ^6.3.1
   lints: ^2.0.0
   test: ^1.14.3
+  timezone: ^0.9.0 # Used for testing only
```

Comment:
```suggestion
  timezone: ^0.9.0
```

Reply:
if we need it as dev_dep then we need it. I suggest we don't have the comment here